website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Operator Observability Best Practices"
+linkTitle: "Observability Best practices"
+weight: 6
+description: This guide describes the best practices concepts for adding Observability to operators.
+---
+
+## Operator Observability Best Practices
+
+In this document we will outline what operators require in order to meet the "Deep Insights" capability level and provide best practices and examples for creating metrics, recording rules and alerts. It is based on the general guidelines in [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/).
+
+**Note:** For technical documentation of how to add metrics to your operator, please read the [Metrics](https://book.kubebuilder.io/reference/metrics.html) section of the Kubebuilder documentation.
+
+### Deep Insights capability level requirements
+
+1. **Health and Performance metrics** for all of the operator components - Implemented based on the guidelines below.
+2. **Metrics Documentation** - All metrics should have documentation.
+3. **Alerts** for when things are not working as expected for each of the operator's components - Implemented based on the guidelines below.
+4. **Alerts Runbooks** - Each alert MUST include a runbook_url annotation and a runbook.
+5. **Alerts and Metrics Tests** - E2E Testing for metrics and alerts and unit tests for alerts.
+6. **Events** - Custom Resources MUST emit custom events for the operations taking place.
+7. **Metering** - Operator leverages Operator Metering.
```

Comment:
```suggestion
```

We can recommend/suggest using this solution to achieve some goal but it is not a pre-requirement to achieve the level. Also, why use this solution would be a recommendation? What advantages it can bring? What complexities to achieve the level it solves?

Reply:
@camilamacedo86 +1. Ill add this info